src/getFilePath.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs').promises;
+const path = require('path');
+
+const readFileByPath = async (requestedPath) => {
+  const fileName = requestedPath.replace(/^file\/?/, '') || 'index.html';
```

Comment:
The regular expression should ensure that the path starts with `/file/`. Consider updating it to `/^file\//` to ensure correct path handling.